.github/workflows/tests.yml

```
@@ -0,0 +1,25 @@
+name: test_tox
+
+on:
+  - push
+  - pull_request
```

Якщо не помиляюсь, то одно `push` буде достатньо для усього простого.

```suggestion
  - push
```

---

оукей